README.md

```diff
@@ -75,16 +75,58 @@ spark users can help us in finding what is missing here by raising challenging t
 
 ### How to use Sparklens? ###
 
-Using the sparklens package.
+####1. Using the sparklens package while running your app
 
+Use the following arguments in spark-submit or spark-shell:
 ```
---packages qubole:sparklens:0.1.2-s_2.11
+--packages qubole:sparklens:0.1.3-s_2.11
 --conf spark.extraListeners=com.qubole.sparklens.QuboleJobListener
 ```
 
-Or 
+####2. Run from sparklens-dump
 
-Checkout the code and use the normal sbt commands: 
+You can choose not to run sparklens inside the app, but at a later time. Run you app as above 
+with an additional conf:
+```
+--packages qubole:sparklens:0.1.3-s_2.11
+--conf spark.extraListeners=com.qubole.sparklens.QuboleJobListener
+--conf spark.sparklens.simulation.async=true
```

Comment:
When not running, can we print a line saying that a dump has been saved for later usage.

Reply:
Done.